lib/stats/folder.go

```diff
@@ -35,45 +35,71 @@ func NewFolderStatisticsReference(ldb *db.Lowlevel, folder string) *FolderStatis
 	}
 }
 
-func (s *FolderStatisticsReference) GetLastFile() LastFile {
-	at, ok := s.ns.Time("lastFileAt")
-	if !ok {
-		return LastFile{}
+func (s *FolderStatisticsReference) GetLastFile() (LastFile, error) {
+	at, ok, err := s.ns.Time("lastFileAt")
+	if err != nil {
+		return LastFile{}, err
+	} else if !ok {
+		return LastFile{}, nil
 	}
-	file, ok := s.ns.String("lastFileName")
-	if !ok {
-		return LastFile{}
+	file, ok, err := s.ns.String("lastFileName")
+	if err != nil {
+		return LastFile{}, err
+	} else if !ok {
+		return LastFile{}, nil
+	}
+	deleted, _, err := s.ns.Bool("lastFileDeleted")
+	if err != nil {
+		return LastFile{}, err
+	} else if !ok {
```

Comment:
This check is redundant, unless the `ok` shouldn't be omitted in the last assignment above (also if it was purposely omitted, maybe add a comment - I don't understand why it might be).

Reply:
That's a copy-paste-o